one/one/main.cpp

```diff
@@ -0,0 +1,137 @@
+#include <iostream>
+using namespace std;
+
+#define endl '\n';
+
+void TwoOne() {
+    const char* msg = "Hello World";
+    cout << msg << endl;
+}
+
+void TwoTwo() {
+    const int n = 5;
```

Comment:
👍 fyi the `const int n = 5` is the You.i preferred method of declaring constants (instead of the `#define endl '\n';` from the MIT guide)